cartridges/int_turnto_sitegenesis_v5/cartridge/static/default/js/teasers.js

```diff
@@ -8,38 +8,38 @@
  * @function
  * @description The loadTeaserCounts function first loads the UGC counts maintained by TurnTo and then calls the ​populateTeaser​ function.
  * @name loadTeaserCounts
- * @param {String} sku the product ID 
+ * @param {String} sku the product ID
```

Comment:
Are we keeping this teaser code around to support clients already using this code?
I'm guessing so since this appears to be for legacy saleforce (site genesis).

Reply:
I need to dig into this more when I have time.